Client/mods/deathmatch/logic/CStaticFunctionDefinitions.cpp

```diff
@@ -2589,6 +2589,9 @@ bool CStaticFunctionDefinitions::SetPedOnFire(CClientEntity& Entity, bool bOnFir
 {
     if (IS_PED(&Entity))
     {
+        if (!Entity.IsLocalEntity())
+            return false;
+
```

Comment:
Isn't this going to restrict `setPedOnFire`?

Reply:
Together with Dutchman, we concluded that this approach would be better to prevent Lua executors from igniting everything around by calling these functions on the client side